packages/headless/src/features/facets/category-facet-set/category-facet-set-slice.test.ts

```diff
@@ -114,6 +113,48 @@ describe('category facet slice', () => {
     expect(finalState).toEqual(categoryFacetSet);
   });
 
+  describe('#restoreSearchParameters', () => {
```

Comment:
What happens when someone enters an invalid path (for example if they miss the last character when copy & pasting), does it ignore the incorrect value or does it cause the facet to stop working entirely?

Reply:
The facet would work normally and reflect the invalid path restored from the url. Since the path is invalid, there would be no results for the selection. A user could get out of that state by deselecting all values on the category facet.

We would not be able to detect invalid values in a url to ignore them without making additional calls to the index via the search-api.

Reply:
Edit: I just discovered that if the **first** value of the path is invalid, then the category stops working and it is not possible to deselect, and the page is unusable. Good catch, I will look into this!